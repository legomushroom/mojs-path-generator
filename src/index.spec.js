import React from "react";
import MojsPathGenerator, {
  MojsPathGenerator as NamedGenerator
} from "./index";
import renderer from "react-test-renderer";

test("Component should mount", () => {
  const component = renderer.create(<MojsPathGenerator options={{ a: 1 }} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Named component should mount", () => {
  const component = renderer.create(<NamedGenerator options={{ a: 1 }} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
