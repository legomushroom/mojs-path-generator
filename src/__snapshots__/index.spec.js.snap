exports[`test Component should mount 1`] = `
<div>
  <svg
    height="500"
    width="500">
    <path
      d="M150 0 L75 200 L225 200 Z"
      fill="#000"
      stroke="red"
      strokeWidth="3" />
  </svg>
</div>
`;

exports[`test Named component should mount 1`] = `
<div>
  <svg
    height="500"
    width="500">
    <path
      d="M150 0 L75 200 L225 200 Z"
      fill="#000"
      stroke="red"
      strokeWidth="3" />
  </svg>
</div>
`;
